docs/tutorial/tutorial-5/android.rst

```diff
@@ -2,9 +2,382 @@
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/bin/javac" -version
+      javac 1.8.0_242
+
+    If you get a "Command not found" error, look for the ``export`` command in
+    the previous section, and run it.
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      $ "$JAVA_HOME/javac" -version
+      javac 1.8.0_242
+
+    If you get a "Command not found" error, look for the ``export`` command in
+    the previous section, and run it.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>"%JAVA_HOME%\bin\javac" -version
+      javac 1.8.0_242
+
+    If you get a "Command not found" error, re-run the installer from AdoptOpenJDK
+    and be sure to configure the installer to "Set JAVA_HOME variable".
+
+Create an Android app and compile it
+====================================
+
+The process of deploying an application to Android is very similar to the
+process for deploying as a desktop application. First, you run the ``create``
+command. This downloads an Android app template and adds your Python code to it.
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (beeware-venv) $ briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+  .. group-tab:: Windows
+
+    .. code-block:: doscon
+
+      C:\...>briefcase create android
+
+      [helloworld] Generating application template...
+      Using app template: https://github.com/beeware/briefcase-android-gradle-template.git
+      ...
+      [helloworld] Installing support package...
+      ...
+      [helloworld] Installing dependencies...
+      ...
+      [helloworld] Installing application code...
+      ...
+      [helloworld] Installing application resources...
+      ...
+      [helloworld] Application created.
+
+Once this completes, we'll now have an ``android`` directory in your project.
+This directory will contain a ``Hello World`` folder, which will contain an
+Android project configured to use gradle, your application code, and a support
```

Comment:
"Android project configured to use gradle, your application code..." -> "Android project with a Gradle build configuration. This project will contain your application code..."